backend-api/src/functions/asyncActiveSession/jwe/tests/jweDecrypter.test.ts

```diff
@@ -58,7 +58,7 @@ describe("Decrypt JWE", () => {
       expect(result.isError).toBe(true);
       expect(result.value).toStrictEqual({
         errorMessage:
-          "Error decrypting JWE: Unable to decrypt payload: Some mock symmetric decryption error",
+          "Error decrypting JWE: Unable to decrypt payload: Error: Some mock symmetric decryption error",
```

Comment:
I need to format these errors a bit better but again I didn't want to do this in this PR and make it even bigger

Reply:
Already done - resolving